sci-libs/openblas/openblas3-0.3.1.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="An optimized BLAS library"
+DESCRIPTION="OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD \
+version."
+HOMEPAGE="http://www.openblas.net/"
+COPYRIGHT="2011-2018 The OpenBLAS Project"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="http://github.com/xianyi/OpenBLAS/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="1f5e956f35f3acdd3c74516e955d797a320c2e0135e31d838cbdb3ea94d0eb33"
+SOURCE_FILENAME="OpenBLAS-$portVersion.tar.gz"
+SOURCE_DIR="OpenBLAS-$portVersion"
+PATCHES="openblas3-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	openblas3$secondaryArchSuffix = $portVersionCompat
+	lib:libopenblas$secondaryArchSuffix = $libVersionCompat
+	lib:libopenblasp_r$portVersion$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgfortran$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	openblas3${secondaryArchSuffix}_devel = $portVersionCompat
+	devel:libopenblas$secondaryArchSuffix = $libVersionCompat
+	devel:libopenblasp_r$portVersion$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	openblas3$secondaryArchSuffix == $portVersion base
+	"
+CONFLICTS_devel="
+	openblas2${secondaryArchSuffix}_devel
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gfortran$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:perl
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage openblas3$secondaryArchSuffix \
```

Comment:
```diff
-defineDebugInfoPackage openblas3$secondaryArchSuffix \
+defineDebugInfoPackage openblas$secondaryArchSuffix \
```

Also, don't forget to rename the files:
openblas**3**-0.3.1.{recipe,patchset} :arrow_right: openblas-0.3.1.{recipe,patchset}.